common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -82,14 +82,16 @@ steps:
     env:
       RUSH_BUILD_CACHE_CREDENTIAL: $(RushBuildCacheSAS)
       RUSH_BUILD_CACHE_ENABLED: ${{parameters.rushBuildCacheEnabled}}
+      VITE_CI: true
     condition: and(succeeded(), ne(variables['Agent.OS'], 'Linux'))
-  # TEMP: build on linux but with 50% CPU saturation to prevent OOM
-  - script: node common/scripts/install-run-rush.js build -v -p '50%'
+
+  - script: node common/scripts/install-run-rush.js build -v -p max
```

Comment:
do we need this separate task now? cant we revert the addition form your last pr and just set this new env var?

Reply:
You're right, I've removed the separate task.